modules/googletest/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+zephyr_library()
+
+# Disable use of pthreads in GoogleTest
+set(gtest_disable_pthreads ON CACHE BOOL "Build GoogleTest without Pthread")
+add_definitions(-DGTEST_HAS_FILE_SYSTEM=0)
```

Comment:
Don't you want `-DGTEST_OS_ZEPHYR=1` here, since that will set the disable the FS for you?

Reply:
I will check but actually, this should not be required anymore.